tests/variance_broadcast_test.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Missing a test for mutual broadcast without bins:
```python
v1 = sc.ones(sizes={'x': 5}, with_variances=True)
v2 = sc.ones(sizes={'y': 4}, with_variances=True)
v1 * v2
```
and ideally with all permutations of `with_variances` args.

Otherwise I can't think of any other cases right now.

Reply:
I found that `sc.values` (and friend) do not work on broadcast results any more. I suppose we should treat them in the same was as `copy`, i.e., make an exception?

Reply:
Agree. At least for `sc.values`. `sc.variances` is maybe not a good idea?

Reply:
I think I'd make it consistent. `sc.variances` and `sc.stddevs` are pretty specialized (read, rarely used), so I don't see much utility in disabling extraction of variances from broadcast variables.

Reply:
Test still missing?